src/nodes/display/ColorAdjustmentNode.js

```diff
@@ -86,8 +88,8 @@ export const saturation = nodeProxy( ColorAdjustmentNode, ColorAdjustmentNode.SA
 export const vibrance = nodeProxy( ColorAdjustmentNode, ColorAdjustmentNode.VIBRANCE );
 export const hue = nodeProxy( ColorAdjustmentNode, ColorAdjustmentNode.HUE );
 
-export const lumaCoeffs = vec3( 0.2126729, 0.7151522, 0.0721750 ); // // assumes rgb is in linear color space with sRGB primaries and D65 white point
-export const luminance = ( color, luma = lumaCoeffs ) => dot( color, luma );
+const luminanceCoefficients = /*#__PURE__*/ new Vector3();
+export const luminance = ( color, luma = vec3( ... ColorManagement.getLuminanceCoefficients( luminanceCoefficients ) ) ) => dot( color, luma );
```

Comment:
Can we avoid the word `luma`. How about `coeffs`, or preferably, `luminanceCoefficients`?

Reply:
Done!